js/noms/src/json-convert.js

```diff
@@ -13,7 +13,7 @@ import {
 import type Value from './value.js';
 
 type JSON = string | number | boolean | null | JSONObject | JSONArray;
-type JSONObject = { [key:string]: JSON };
+type JSONObject = { [key: string]: JSON };
```

Comment:
By "this" do you mean 3.7.1? Not sure what your comment means exactly :-) anyhow, submitting.